packages/server/src/AccountsServer.js

```diff
@@ -174,6 +174,35 @@ export class AccountsServer {
     return { accessToken, refreshToken };
   }
   async logout(accessToken: string): Promise<void> {
+    const session : SessionType = await this.findSessionByAccessToken(accessToken);
+    if (session.valid) {
+      const user = await this.db.findUserById(session.userId);
+      if (!user) {
+        throw new AccountsError({
+          message: 'User not found',
+        });
+      }
+      await this.db.invalidateSession(session.sessionId);
+    } else { // eslint-disable-line no-else-return
```

Comment:
I think the lifespan of expired sessions should be configurable. It can provide a useful log for user account activity. Keep the latest 10 sessions for example.